Automattic/newspack-plugin
# DRAFT Ad Management Wizard

### All Submissions:


### Changes proposed in this Pull Request:

Provides a simple wizard for setting up and managing placement of ads. Users can select their chosen ad network from an initial set of three. Two of the networks then allow for choosing pre-defined placements, borrowing from the Jetpack Ads set up. The third (AdSense) uses SiteKit.

This initial draft includes the UI elements that can be stepped through as the user would do, but with no backend functionality to make the changes. This PR is to invite comment and review on the current composition of the wizard and the user experience.

Closes #82

### How to test the changes in this Pull Request:

1. Apply the patch
2. Navigate to Newspack > Advertising
3. Select one of the ad networks and observe the set up flow.

### Other information:
